ZWaveLib/Values/CentralSceneValue.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ZWaveLib.Values
+{
+    public enum CentralScenePressType
+    {
+        Pressed1times = 0x0,
+        Released = 0x1,
+        HeldDown = 0x2,
+        Pressed2times = 0x3,
```

Comment:
Please, rename this and following enum values to Pressed2**T**imes.
Also, please be consistent in using byte values (you are mixing 0x3 with 0x**0**3 style). Let it be 0x**0**3.